helix-query.yaml

```diff
@@ -226,3 +226,78 @@ indices:
       lastModified:
         select: none
         value: parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
+  hk:
+    include:
+      - /hk/**
+    exclude:
+      - '**/gnav/**'
+      - '**/drafts/**'
+      - '**/fragments/**'
+      - '**/localization/**'
+    target: /hk/dc-shared/assets/sitemaps/query-index-hk.xlsx
```

Comment:
The xlsx path is not consistent with the existing locales. There are hk_en and hk_zh in SharePoint